setup.cfg

```diff
@@ -0,0 +1,28 @@
+[metadata]
+name = kingpin
+version = attr: kingpin.version.__version__
+url = https://github.com/Nextdoor/kingpin
+download_url= http://pypi.python.org/pypi/kingpin#downloads
+author = Nextdoor Engineering
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Operating System :: POSIX
+    Programming Language :: Python :: 3
+    Topic :: Software Development
+license = Apache License, Version 2.0
+description = Deployment Automation Engine
+long_description = file: README.rst
+keywords = deployment, automation, kingpin, aws
+
+[options]
+install_requires = file: requirements.txt
+python_requires = ~=3.7
```

Comment:
For the future: I believe Python 3.6 recently lost support, so it may eventually be a project to get this up to Python 3.11 or whatever. Until then, thanks for undertaking this effort in general!